include/TGUI/Container.hpp

```diff
@@ -107,18 +107,26 @@ namespace tgui
             return m_widgets;
         }
 
-
+#ifndef TGUI_REMOVE_DEPRECATED_CODE
         /////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
         /// @brief Returns a list of the names of all the widgets in this container
         ///
         /// @return Vector of all widget names
         ///
         /////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
-        const std::vector<sf::String>& getWidgetNames() const
+        TGUI_DEPRECATED("Use for(getWidgets())->getWidgetName() instead") const std::vector<sf::String> getWidgetNames() const
```

Comment:
`for(getWidgets())->getWidgetName()` might not be very clear. Maybe something like `Use getWidgets() and Widget::getWidgetName instead` as description?